docs/src/main/sphinx/connector/snowflake.md

```diff
@@ -49,31 +49,54 @@ multiple instances of the Snowflake connector.
 
 ## Type mapping
```

Comment:
Please use the same wording and also source syntax (list-table) as in the PostgreSQL connector docs. 

See https://trino.io/docs/current/connector/postgresql.html#type-mapping and 
https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/connector/postgresql.md#type-mapping